components/dialog/index.css

```diff
@@ -11,377 +11,429 @@ governing permissions and limitations under the License.
-  justify-content: flex-end;
+	/* this padding should be safe as button group is always end aligned */
+	padding-inline-start: var(--mod-standard-dialog-spacing-grid-padding, var(--spectrum-standard-dialog-spacing-grid-padding));
 
-  /* this padding should be safe as button group is always end aligned */
-  padding-inline-start: var(--mod-dialog-confirm-gap-size, var(--spectrum-dialog-confirm-gap-size));
+	&.spectrum-Dialog-buttonGroup--noFooter {
+		grid-area: footer-start / footer-start / buttonGroup-end / buttonGroup-end;
+	}
 
-  &.spectrum-Dialog-buttonGroup--noFooter {
-    grid-area: footer-start / footer-start / buttonGroup-end / buttonGroup-end;
-  }
+	/* overrides ButtonGroup's justify-content: normal */
+	.spectrum-ButtonGroup {
+		justify-content: flex-end;
+	}
 }
 
-.spectrum-Dialog.spectrum-Dialog--dismissable .spectrum-Dialog-grid {
-  grid-template-columns:
-    var(--mod-dialog-confirm-padding-grid, var(--spectrum-dialog-confirm-padding-grid)) auto 1fr auto minmax(0,  auto)
-    minmax(0,  var(--mod-dialog-confirm-close-button-size, var(--spectrum-dialog-confirm-close-button-size))) var(--mod-dialog-confirm-padding-grid, var(--spectrum-dialog-confirm-padding-grid));
-  grid-template-rows: auto var(--mod-dialog-confirm-padding-grid, var(--spectrum-dialog-confirm-padding-grid)) auto auto 1fr auto var(
-      --mod-dialog-confirm-padding-grid, var(--spectrum-dialog-confirm-padding-grid)
-    );
-  grid-template-areas:
-    "hero hero    hero    hero        hero        hero        hero"
-    ".    .       .       .           .           closeButton closeButton"
-    ".    heading header  header      typeIcon    closeButton closeButton"
-    ".    divider divider divider     divider     divider     ."
-    ".    content content content     content     content     ."
-    ".    footer  footer  buttonGroup buttonGroup buttonGroup ."
-    ".    .       .       .           .           .           .";
-
-  .spectrum-Dialog-buttonGroup {
-    display: none;
-  }
-
-  .spectrum-Dialog-footer {
-    grid-area: footer / footer/ buttonGroup / buttonGroup;
-    color: var(--mod-dialog-confirm-description-text-color, var(--spectrum-dialog-confirm-description-text-color));
-  }
+/* Dismissible Dialog (with close button) */
+.spectrum-Dialog--dismissable .spectrum-Dialog-grid {
+	grid-template-columns:
+		var(--mod-standard-dialog-spacing-grid-padding, var(--spectrum-standard-dialog-spacing-grid-padding))
+		auto
+		1fr
+		auto
+		minmax(0, auto)
+		minmax(0, calc(var(--spectrum-standard-dialog-spacing-edge-to-content) - var(--spectrum-standard-dialog-spacing-edge-to-close-button)))
+		var(--mod-standard-dialog-spacing-grid-padding, var(--spectrum-standard-dialog-spacing-grid-padding));
+	grid-template-rows:
+		auto
+		var(--mod-standard-dialog-spacing-grid-padding, var(--spectrum-standard-dialog-spacing-grid-padding))
+		auto
+		1fr
+		auto
+		var(--mod-standard-dialog-spacing-grid-padding, var(--spectrum-standard-dialog-spacing-grid-padding));
```

Comment:
I can definitely change this formatting back- this change wasn't necessary. I was just having a hard time reading them all listed out horizontally.

Reply:
I think it looks a little easier to read this way too!